src/domUpdates.js

```diff
@@ -1,6 +1,7 @@
 import $ from 'jquery';
 import Customer from './customer.js'
 import {hideOrShowElement} from './index.js';
+import {giveUser} from './index.js';
```

Comment:
I may be wrong, but I don't think you need the curly braces here or on line 3.

Reply:
Not sure about that one. I will look into it. Thanks for brining it to my attention.